src/phoenix/server/api/types/DatasetExample.py

```diff
@@ -47,3 +55,31 @@ async def span(
             if (span := await info.context.data_loaders.dataset_example_spans.load(self.id_attr))
             else None
         )
+
+    @strawberry.field
+    async def experiments(
+        self,
+        info: Info[Context, None],
+        first: Optional[int] = 50,
+        last: Optional[int] = UNSET,
+        after: Optional[CursorString] = UNSET,
+        before: Optional[CursorString] = UNSET,
+    ) -> Connection[Experiment]:
+        args = ConnectionArgs(
+            first=first,
+            after=after if isinstance(after, CursorString) else None,
+            last=last,
+            before=before if isinstance(before, CursorString) else None,
+        )
+        example_id = self.id_attr
+        query = (
+            select(models.Experiment)
+            .join(models.ExperimentRun, models.Experiment.id == models.ExperimentRun.experiment_id)
+            .where(models.ExperimentRun.dataset_example_id == example_id)
+            .order_by(models.Experiment.id.asc())
```

Comment:
probably want descending
```suggestion
            .order_by(models.Experiment.id.desc())
```

Reply:
addressed in 5e654faec0ac829a8a08858ff85e7693ad99b917